src/transport/SecureSessionMgr.cpp

```diff
@@ -92,136 +92,157 @@ CHIP_ERROR SecureSessionMgr::SendMessage(NodeId peerNodeId, System::PacketBuffer
 CHIP_ERROR SecureSessionMgr::SendMessage(PayloadHeader & payloadHeader, NodeId peerNodeId, System::PacketBufferHandle msgBuf,
                                          EncryptedPacketBufferHandle * bufferRetainSlot)
 {
-    return SendMessage(payloadHeader, peerNodeId, std::move(msgBuf), bufferRetainSlot, false);
+    PacketHeader packetHeader;
```

Comment:
This empty packet header required to call a method seems a bit odd.

we could call it unusedParsedPacketHeader to make it clearer why it exists, however if we find a way to avoid it it would be even nicer.